### PR TITLE
[FR] Update `People/Community_Contributors`

### PR DESCRIPTION
**Pull request by Pisapou**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/People/Community_Contributors/fr.md [hunk 1]
```diff
@@ -192,3 +192,27 @@ Les **contributeurs de la communauté** sont les utilisateurs ayant fourni des s
...
+
+## 2022
+
+### Juin
+
+*Pour l'article du news, voir : [Community Contributors: 2021](https://osu.ppy.sh/home/news/2022-06-30-community-contributors-2021)*
+
+| Utilisateur | Contributions |
+| :-- | :-- |
+| ::{ flag=FR }:: [Kasumi-sama](https://osu.ppy.sh/users/6177263) | Contribution exceptionnelle à la scène des tournois de la communauté osu!taiko |
```
Comment (Pisapou) [inline wiki/People/Community_Contributors/fr.md:204]:
```suggestion
| ::{ flag=FR }:: [Kasumi-sama](https://osu.ppy.sh/users/6177263) | Contribution exceptionnelle aux tournois de la communauté osu!taiko |
```

File: wiki/People/Community_Contributors/fr.md [hunk 2]
```diff
@@ -192,3 +192,27 @@ Les **contributeurs de la communauté** sont les utilisateurs ayant fourni des s
...
+
+| Utilisateur | Contributions |
+| :-- | :-- |
+| ::{ flag=FR }:: [Kasumi-sama](https://osu.ppy.sh/users/6177263) | Contribution exceptionnelle aux tournois de la communauté osu!taiko |
+| ::{ flag=SA }:: [frenzibyte](https://osu.ppy.sh/users/14210502) | Contribution exceptionnelle au développement d'osu!(lazer) |
+| ::{ flag=HR }:: [Susuko3](https://osu.ppy.sh/users/18945305) | Contribution exceptionnelle au développement d'osu!(lazer) et d'osu!framework |
+| ::{ flag=RU }:: [StanR](https://osu.ppy.sh/users/7217455) | Contribution exceptionnelle au développement et à la maintenance des points de performance d'osu!|
+| ::{ flag=GB }:: [Apo11o](https://osu.ppy.sh/users/9558549) | Contribution exceptionnelle au développement des points de performance d'osu! |
+| ::{ flag=AU }:: [MBmasher](https://osu.ppy.sh/users/4498616) | Contribution exceptionnelle au développement des points de performance d'osu! et à la revitalisation du mod Flashlight |
+| ::{ flag=SE }:: [Walavouchey](https://osu.ppy.sh/users/5773079) | Contribution exceptionnelle au projet wiki d'osu! |
```
Comment (Ganondorf-1) [inline wiki/People/Community_Contributors/fr.md:205]:
```suggestion
| ::{ flag=SE }:: [Walavouchey](https://osu.ppy.sh/users/5773079) | Contribution exceptionnelle au projet osu! wiki |
```

File: wiki/People/Community_Contributors/fr.md [hunk 3]
```diff
@@ -192,3 +192,27 @@ Les **contributeurs de la communauté** sont les utilisateurs ayant fourni des s
...
+### Juin
+
+*Pour l'article du news, voir : [Community Contributors: 2021](https://osu.ppy.sh/home/news/2022-06-30-community-contributors-2021)*
+
+| Utilisateur | Contributions |
+| :-- | :-- |
+| ::{ flag=FR }:: [Kasumi-sama](https://osu.ppy.sh/users/6177263) | Contribution exceptionnelle aux tournois de la communauté osu!taiko |
+| ::{ flag=SA }:: [frenzibyte](https://osu.ppy.sh/users/14210502) | Contribution exceptionnelle au développement d'osu!(lazer) |
+| ::{ flag=HR }:: [Susuko3](https://osu.ppy.sh/users/18945305) | Contribution exceptionnelle au développement d'osu!(lazer) et d'osu!framework |
+| ::{ flag=RU }:: [StanR](https://osu.ppy.sh/users/7217455) | Contribution exceptionnelle au développement et à la maintenance des points de performance d'osu!|
```
Comment (Ganondorf-1) [inline wiki/People/Community_Contributors/fr.md:202]:
```suggestion
| ::{ flag=RU }:: [StanR](https://osu.ppy.sh/users/7217455) | Contribution exceptionnelle au développement et à la maintenance des points de performance d'osu! |
```

File: wiki/People/Community_Contributors/fr.md [hunk 4]
```diff
@@ -192,3 +192,27 @@ Les **contributeurs de la communauté** sont les utilisateurs ayant fourni des s
...
+| ::{ flag=GB }:: [Apo11o](https://osu.ppy.sh/users/9558549) | Contribution exceptionnelle au développement des points de performance d'osu! |
+| ::{ flag=AU }:: [MBmasher](https://osu.ppy.sh/users/4498616) | Contribution exceptionnelle au développement des points de performance d'osu! et à la revitalisation du mod Flashlight |
+| ::{ flag=SE }:: [Walavouchey](https://osu.ppy.sh/users/5773079) | Contribution exceptionnelle au projet wiki d'osu! |
+| ::{ flag=ID }:: [Niva](https://osu.ppy.sh/users/197805) | Contribution exceptionnelle au projet wiki d'osu! |
```
Comment (Ganondorf-1) [inline wiki/People/Community_Contributors/fr.md:206]:
```suggestion
| ::{ flag=ID }:: [Niva](https://osu.ppy.sh/users/197805) | Contribution exceptionnelle au projet osu! wiki |
```

File: wiki/People/Community_Contributors/fr.md [hunk 5]
```diff
@@ -192,3 +192,27 @@ Les **contributeurs de la communauté** sont les utilisateurs ayant fourni des s
...
+| ::{ flag=NZ }:: [Technocoder](https://osu.ppy.sh/users/10338558) | Contribution exceptionnelle au support technique de macOS |
+| ::{ flag=LT }:: [huu](https://osu.ppy.sh/users/6044237) | Contribution exceptionnelle à la gestion et à l'organisation de Project Loved |
```
Comment (Ganondorf-1) [inline wiki/People/Community_Contributors/fr.md:208]:
```suggestion
| ::{ flag=LT }:: [huu](https://osu.ppy.sh/users/6044237) | Contribution exceptionnelle à la gestion et à l'organisation du Project Loved |
```

File: wiki/People/Community_Contributors/fr.md [hunk 6]
```diff
@@ -192,3 +187,27 @@ Les **contributeurs de la communauté** sont les utilisateurs ayant fourni des s
...
+| ::{ flag=NL }:: [OliBomby](https://osu.ppy.sh/users/6573093) | Contribution exceptionnelle au mapping d'osu! via le développement d'un utilitaire |
+| ::{ flag=TR }:: [frukoyurdakul](https://osu.ppy.sh/users/7612550) | Contribution exceptionnelle au mapping osu!taiko via le développement d'un utilitaire |
+| ::{ flag=BR }:: [LeoFLT](https://osu.ppy.sh/users/3668779) | Contribution exceptionnelle à la scène des tournois osu! et aux World Cups |
+| ::{ flag=US }:: [ChillierPear](https://osu.ppy.sh/users/9501251) | Contribution exceptionnelle à la scène des tournois osu! et aux World Cups |
+| ::{ flag=NL }:: [cavoeboy](https://osu.ppy.sh/users/7361815) | Contribution exceptionnelle aux évènements osu! IRL et aux tournois |
```
Comment (Ganondorf-1) [inline wiki/People/Community_Contributors/fr.md:213]:
```suggestion
| ::{ flag=NL }:: [cavoeboy](https://osu.ppy.sh/users/7361815) | Contribution exceptionnelle aux évènements IRL d'osu! et aux tournois |
```

**Review by Soetch — COMMENTED** [review]
Good translation after some trouble finding the good word between "exceptionnelle" and "remarquable".

**Review by Ganondorf-1 — CHANGES_REQUESTED** [review]

File: wiki/People/Community_Contributors/fr.md [hunk 7]
```diff
@@ -192,3 +187,27 @@ Les **contributeurs de la communauté** sont les utilisateurs ayant fourni des s
...
+| ::{ flag=ID }:: [Niva](https://osu.ppy.sh/users/197805) | Contribution exceptionnelle au projet osu! wiki |
+| ::{ flag=NZ }:: [Technocoder](https://osu.ppy.sh/users/10338558) | Contribution exceptionnelle au support technique de macOS |
+| ::{ flag=LT }:: [huu](https://osu.ppy.sh/users/6044237) | Contribution exceptionnelle à la gestion et à l'organisation du Project Loved |
+| ::{ flag=NL }:: [OliBomby](https://osu.ppy.sh/users/6573093) | Contribution exceptionnelle au mapping d'osu! via le développement d'un utilitaire |
+| ::{ flag=TR }:: [frukoyurdakul](https://osu.ppy.sh/users/7612550) | Contribution exceptionnelle au mapping osu!taiko via le développement d'un utilitaire |
+| ::{ flag=BR }:: [LeoFLT](https://osu.ppy.sh/users/3668779) | Contribution exceptionnelle à la scène des tournois osu! et aux World Cups |
+| ::{ flag=US }:: [ChillierPear](https://osu.ppy.sh/users/9501251) | Contribution exceptionnelle à la scène des tournois osu! et aux World Cups |
+| ::{ flag=NL }:: [cavoeboy](https://osu.ppy.sh/users/7361815) | Contribution exceptionnelle aux évènements IRL d'osu! et aux tournois |
```
Comment (Pisapou) [inline wiki/People/Community_Contributors/fr.md:213]:
```suggestion
| ::{ flag=NL }:: [cavoeboy](https://osu.ppy.sh/users/7361815) | Contribution exceptionnelle aux évènements IRL d'osu! et aux tournois |

```

File: wiki/People/Community_Contributors/fr.md [hunk 8]
```diff
@@ -192,3 +187,27 @@ Les **contributeurs de la communauté** sont les utilisateurs ayant fourni des s
 | ::{ flag=DE }:: [RockRoller](https://osu.ppy.sh/users/8388854) | Contribution exceptionnelle à la scène de skinning et de modding d'osu! |
 | ::{ flag=US }:: [I Must Decrease](https://osu.ppy.sh/users/2773526) | Contribution exceptionnelle à l'entretien et au développement des scores |
 | ::{ flag=US }:: [this1neguy](https://osu.ppy.sh/users/1797189) | Contribution exceptionnelle aux World Cups et aux tournois communautaires |
+
+## 2022
+
+### Juin
+
+*Pour l'article du news, voir : [Community Contributors: 2021](https://osu.ppy.sh/home/news/2022-06-30-community-contributors-2021)*
```
Comment (Ganondorf-1) [inline wiki/People/Community_Contributors/fr.md:195]:
```suggestion
*Pour l'article de news, voir : [Community Contributors: 2021](https://osu.ppy.sh/home/news/2022-06-30-community-contributors-2021)*
```

**Review by Ganondorf-1 — CHANGES_REQUESTED** [review]

**Review by Ganondorf-1 — APPROVED** [review]

**Review by TPGPL — APPROVED** [review]

**Pull request merged**